Author: Andreas Tille <tille@debian.org>
Date: Wed, 25 Apr 2012 22:31:55 +0200
Bug-Closed: http://bugs.debian.org/667161
Description: Build with g++ version 4.7
 Follow the hint given at
   http://gcc.gnu.org/gcc-4.7/porting_to.html
 which was linked in the bug report and add a missing
 include.

--- fastx-toolkit.orig/src/libfastx/sequence_alignment.h
+++ fastx-toolkit/src/libfastx/sequence_alignment.h
@@ -19,6 +19,7 @@
 #define __SEQUENCE_ALIGNMENT_HEADER__
 
 #include <err.h>
+#include <unistd.h>
 
 struct SequenceAlignmentResults
 {
